src/hu_stuff.c

```diff
@@ -885,6 +885,449 @@ static int HU_top(int i, int idx1, int top1)
   return i;
 }
 
+// do the hud ammo display
+static void HU_widget_build_ammo (void)
+{
+  char ammostr[80], *s; //jff 3/8/98 allow plenty room for dehacked mods
```

Comment:
Does anybody know why we need this `ammostr[80]` *and* `hud_ammostr[80]`? And what does this have to do with dehacked?

Reply:
This is even still present in PrBoom+. But why should we carry two equally sized string buffer?

Reply:
Theoretically, it is possible to significantly increase the quantity/max. ammo in DEHACKED, maybe that's why there are these comments. What if we increase size of `hud_ammostr`?

Reply:
> What if we increase size of `hud_ammostr`?

To what, even more than 80 bytes?

Reply:
Oh yes, with an 8x8 font size, there is no point in increasing the buffer size.